### ISSUE TITLE
(Multi)-Combobox - Binding by Path

### ISSUE THREAD

**Issue by timostark** [SAP/ui5-uiveri5#27]
Hi,

First sorry for the amount of "question" issues - I try to identify all controls I am able to test, and I am still struggling every now an than.

Question: I want to test a multi-combo-box via binding or key value.

For a Select, which is using a **sap.ui.core.Item** that is really simple. Example: https://sapui5.hana.ondemand.com/#/sample/sap.m.sample.Select/preview
I can select by binding path and by key property, and by that have language independent selectors.
```
//context
var item = element(by.control( { controlType : "sap.ui.core.Item", bindingPath: { propertyPath : "ProductId", path : "/ProductCollection/7" } }));

//key (probably better, due to JSON list binding, and by that only "index" access above..
var item = element(by.control( { controlType : "sap.ui.core.Item", properties:[{ key : "HT-1020" }] }));
```

Both Combo-Box and Multi-Combobox is using a **sap.m.StandardListItem** instead. This is a little ugly, as the Standard-List-Item seems to have "outsourced" sap.ui.core.Items (referenced via sap.m.MultiComboBox.prototype._getItemByListItem). In any case - sap.m.StandardListItem only has a title and description attribute - however no binding (probably due to the internal architecture of the popover list).

Example therefore for: https://sapui5.hana.ondemand.com/#/sample/sap.m.sample.ComboBox/preview
The only selector I can think of is selecting by a title - which is of course super annoying, as possibly language dependent.
```
var standardListItem = element(by.control( { controlType : "sap.m.StandardListItem", properties:[{ title : "Bahrain" }] }));
```

Selecting the referenced sap.ui.core.Item does not seem to be possible, as this item does not have a DomRef assigned..

**tldr;** How do you internally select sap.m.StandardListItem from a Combo-Box or a Multi-Combo-Box?

Thanks!

**Comment by maximnaidenov** [comment]
Hi Timo,
I would suggest that you select the items by attribute and prepend a selector for the combobox itself as all items will still have it as a parent. 
BTW: using the RecordReplay.getControlSelectorForElement should automatically do this and always build you the best possible selector.

**Comment by timostark** [comment]
Hi Maxim,

Ok - that is not really perfect, as the attribute might be language dependent. Anyways - It is possible to live with this restriction.

--
Regarding: RecordReplay, I assume you are referring to https://github.com/SAP/openui5/blob/8625992c50e8ab1a4bf1de4553c195ef3e2d49f3/src/sap.ui.core/src/sap/ui/test/RecordReplay.js --> findControlSelectorByDOMElement  ?

Thanks a lot for the hint. This is certainly pretty much duplicated coding, to what I have (just in higher quality probably). For the uiveri5 recording i will probably at least try this method.

Regards,
Timo
